doc/user/bfd.rst

```diff
@@ -518,6 +518,13 @@ You can inspect the current BFD peer status with the following commands:
    frr# show bfd peer 192.168.0.1 json
    {"multihop":false,"peer":"192.168.0.1","id":1,"remote-id":1,"status":"up","uptime":161,"diagnostic":"ok","remote-diagnostic":"ok","receive-interval":300,"transmit-interval":300,"echo-receive-interval":50,"echo-transmit-interval":0,"detect-multiplier":3,"remote-receive-interval":300,"remote-transmit-interval":300,"remote-echo-receive-interval":50,"remote-detect-multiplier":3,"peer-type":"dynamic"}
 
+<<<<<<< Updated upstream
+=======
+If you are running IPV4 BFD Echo, on a Linux platform, we also
```

Comment:
the format is inadequate. please remote '<<<' and '==='.

Reply:
fixed